src/app/chat/views/reactions.nim

```diff
@@ -32,6 +33,11 @@ QtObject:
     self.pubKey = status_settings.getSetting[string](Setting.PublicKey, "0x0")
 
   proc messageEmojiReactionId(self: ReactionView, chatId: string, messageId: string, emojiId: int): string =
+    let chat = self.status.chat.channels[chatId]
+    var chatId = chatId
+    if chat.chatType == ChatType.Profile:
+      chatId = status_utils.getTimelineChatId()
```

Comment:
When reactions are removed, we need to ensure we're removing them from the timeline message list.